### DISCUSSION TITLE
Dihedral parameters in the Martini 3 force field topology file

### DISCUSSION THREAD

**Discussion by yuqingxiong96** [ricalessandri/Martini3-small-molecules#51] [Q&A] [answered]
Hi, in the molecular parameters listed in this document: [,](https://github.com/ricalessandri/Martini3-small-molecules/blob/PMFs-2022RA-ATS/models/martini_v3.0.0_small_molecules_v1.itp) I observed redundancy in the representation of angles and dihedral angles for certain molecules. For instance, in TDMBI molecules, the bond angle 9-10-11 is denoted by both type2 and type10, 
`  9  10  11     10       130.00       25.0  ; [DOI: 10.1021/ct400219n]                                                       9  10  11      2       130.00       50.0  ; [DOI: 10.1021/ct400219n]` 
while the dihedral angle 9-10-11-12 is characterized by 4 times type1.
`   9   10    11    12       1        180.00       1.96  1 ; [DOI: 10.1021/ct400219n]                                          9   10    11    12       1          0.00       0.18  2 ; [DOI: 10.1021/ct400219n]                                          9   10    11    12       1          0.00       0.33  3 ; [DOI: 10.1021/ct400219n]                                          9   10    11    12       1          0.00       0.12  4 ; [DOI: 10.1021/ct400219n] `
The primary distinction lies in their multiplicities. Why then employ four type1 dihedral to depict the same angle? Could these four dihedral angle parameters be derived from QM calculations, each representing a different atom configuration in the AA model?

**Comment by ricalessandri** [comment, marked as answer]
@yuqingxiong96 Sorry for the late reply.

Regarding the angle, that angle uses a "standard" angle potential combined with a "Restricted Bending" potential. The two potentials add up. This will give you a potential similar to Fig 2 of this paper: https://doi.org/10.1021/ct400219n. Note that this is also the paper that introduced the restricted Bending potential so please have a look for what it means. The key point is that it is used for numerical stability of the CG model. 

Similarly, thoss dihedral potentials you point out add up. You can try to plot them and you will see that they will give rise to a more complex dihedral potential, something you couldn't achieve with just one of them. The multiplicity is indeed key. It's the "proper dihedral: periodic type" in Gromacs (https://manual.gromacs.org/current/reference-manual/functions/bonded-interactions.html#proper-dihedrals-periodic-type). I encourage you to plot the function and play with the parameters to get a good feeling for their meaning. Indeed, one way to fit these dihedral potentials is by fitting them to QM data (see also this discussion: https://github.com/ricalessandri/Martini3-small-molecules/discussions/43).
